version/version.go

```diff
@@ -17,3 +17,8 @@ func GetVersion() string {
 
 	return version
 }
+
+// returns jsonrpc representation of the version
+func GetVersionJsonrpc() string {
+	return "v" + Version
```

Comment:
Do we know what kind of response clients like Ganache give for this RPC call?

Reply:
This is up to the client. Polygon network returns this: `{"jsonrpc":"2.0","id":1,"result":"bor/v0.2.13-stable-15965124/linux-amd64/go1.17.6"}`